src/lib/aggregations/get-composite.spec.ts

```diff
@@ -1,13 +1,28 @@
+import { ResponseError } from '@elastic/elasticsearch/lib/errors.js'
+import { Order } from 'lib/enums'
+import { getCompositeSources } from 'lib/utils'
 import { HomeDocument } from 'test/module'
-import { CompositeAggregationSource, getCompositeAggregation } from './get-composite'
+import { setupNestApplication } from 'test/toolkit'
+import { TEST_ELASTICSEARCH_NODE } from 'test/constants'
+import { ElasticsearchService } from 'module/elasticsearch.service'
+import { ElasticsearchModule } from 'module/elasticsearch.module'
+import { getCompositeAggregation } from './get-composite'
 import { getTermsAggregation } from './get-terms'
 
 describe('getCompositeAggregation', () => {
```

Comment:
would be great to have some nested aggregation examples covered as well, its' quite common use case for us. For example top hits aggregation for each composite bucket.